lib/matchers.dart

```diff
@@ -116,7 +119,7 @@ class _CompoundMatcher<T> extends Matcher {
 
   _CompoundMatcher(this._matchers);
 
-  bool matches(T item, Map matchState) {
+  bool matches(covariant T item, Map matchState) {
```

Comment:
from witch SDK is `covariant` available? should update the `pubspec.yaml` accordingly...

Reply:
Good call!